app/controllers/api/v2/permissions_controller.rb

```diff
@@ -34,6 +35,14 @@ def resource_types
         @total = @resource_types.size
         render :resource_types, :layout => 'api/v2/layouts/index_layout'
       end
+
+      private
+
+      def parameter_deprecation
```

Comment:
It doesn't look like this method gets called - perhaps either do it straight from `#index` or use a filter (and check for presence of those params) like other controllers?